ansible/.env.j2

```diff
@@ -0,0 +1,16 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+SECRET_KEY=secret-key
```

Comment:
(Not that it really matters for us right now as it's unused since M1, but worth noting that this is meant to be a secret like the API Token since anyone with this key can tamper with their session cookies)